data/text.json

```diff
@@ -33,16 +36,16 @@
     "warning": "Use the `!register-silph` command in ${bot-channel} to register your Silph Road username."
   },
   "notifications": {
-    "warning": "Use the `!notifications` command in ${bot-channel} to view current notifications for pokémon."
+    "warning": "Use the `!notifications` command in ${bot-channel} to view current notifications for pokÃ©mon."
   },
   "notify": {
-    "warning": "Use the `!want` command in ${bot-channel} to add notifications for pokémon."
+    "warning": "Use the `!want` command in ${bot-channel} to add notifications for pokÃ©mon."
```

Comment:
You’ve got some wonky encoding going on here.

Reply:
What do I need to do about this? This is new territory. Was it because i used notepad the symbols changed?